### DISCUSSION TITLE
load/export specific allsamples/allpatterns on customized path

### DISCUSSION THREAD

**Discussion by weicj** [bangcorrupt/hacktribe#135] [Ideas]
Given E2 has only 26MB ram, usually E2S users would place several allsamples.dll/allpatterns in the SD card on computer, so that we can change different sets quickly. But the problem is, if you modified the sample set and pattern set, you can only save it to the default path. Since E2 does not have such file management tool, that means you have to connect to the computer to move the file, if you want to save it permenantly. I think if there's a function that we can load and export specific allsamples/allpatterns (might be better if we can even rename it) that would be very much helpful.

Actually that's the idea inspired by Novation Circuit Tracks, Circuit has around 100mb ram, and allows you to switch between 'Pack' (with patterns and samples), provide users with really broad workspace.

**Comment by bangcorrupt** [comment]
#95
